README.md

```diff
@@ -16,53 +17,116 @@ When you are done:
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
 # copy & paste your solution here
+def return_max(aList):
+  if len(aList) == 0:
+    return None
+  return max(aList)
 ```
 
 - [Min Value](https://repl.it/@Admin7/minvalue)
 ```python
 # copy & paste your solution here
+def return_min(aList):
+  if len(aList) == 0:
+    return None
+  return min(aList)
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
 # copy & paste your solution here
+def return_extended(aList, iter):
+  for item in iter:
+    aList.append(item)
+  return aList
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
 # copy & paste your solution here
+def create_matrix(h, w):
+  # a list to hold the list
+  aList = []
+  for i in range(h):
+    #make a list for each row
+    bList = []
+    for x in range(w):
+      if w == 1:
+        break
+      else:
+        bList.append('#')
+    aList.append(bList)
+  return aList
 ```
 
 
 #### Dictionaries
 - [Receipt Tally](https://repl.it/@Admin7/receipttally)
 ```python
 # copy & paste your solution here
+def get_total(h, w):
```

Comment:
This was my mistake, but don't be afraid to rename parameters to make them more descriptive.